tasks/elasticsearch-config.yml

```diff
@@ -86,8 +79,8 @@
 - name: Delete Default Logging File
   file: dest=/etc/elasticsearch/logging.yml state=absent
 
-- name: Delete Default Logging File (5.x)
+- name: Delete Default Logging File
   file: dest=/etc/elasticsearch/log4j2.properties state=absent
 
-- name: Delete Default JVM Options File (5.x)
+- name: Delete Default JVM Options File
```

Comment:
Hello!

I do not see that customized jvm.options was on its default location, as required documentation, #235 and [my commit](https://github.com/dekhtyarev/ansible-elasticsearch/commit/1efdbc8052d786bf92b9a18e07b07cb21f809d8e) .

Could you fix it?